### ISSUE TITLE
Installation will fail if databases is not SQLWATCH. Fix hardcoded reference to SQLWATCH database in fn_get_missing_indexes in Power Query

### ISSUE THREAD

**Issue by marcingminski** [marcingminski/sqlwatch#74]

**Comment by jarole** [comment]
Furthermore, I got this error when using database other than SQLWATCH

```
Warning SQL72017: The unnamed foreign key between tables [dbo].[sql_perf_mon_snapshot_header] and
[dbo].[sql_perf_mon_config_snapshot_type] will be checked by checking the table.  Name the foreign key to avoid
checking the entire table and only check the new constraint.
```

**Comment by marcingminski** [comment]
fixed in 9b8a0bf1419a79180fec6c5d6e04b2fa2c7e1095
